website/src/models/group.js

```diff
@@ -352,6 +354,77 @@ GroupSchema.methods.toJSON = function() {
   return doc;
 };
 
+GroupSchema.statics.removeUserFromChallenges = function(user, group, keep, cb) {
```

Comment:
@TheHollidayInn can we change the name of the function since it doesn't only remove the user from challenges but also from active quest & party?